### PR TITLE
PJ_SIM_SUNBNB_2023-14 [FE] Home Page markup 

### PR DESCRIPTION
**Pull request by andreenriquez11**
# Issue Link
https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-14

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1691044119171259

# Definition of Done
* [x] Create a markup for the home page for logged in clients with the components:
    * [x] Home Page Banner
    * [ ] Listings Item Component
    * [x] Cities Component

# Notes
- Will add Listings Item once its merged

# Pre-conditions
none

# Expected Output
image

# Screenshots/Recordings
image
image

### CODE REVIEW

File: fe/src/components/Cities/CityCard.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <v-container fluid>
+      <v-layout align-start row fill-height>
+        <div v-for="city in cities" :key="city.id" class="pl-3 pr-3">
```
Comment (LennardUy104) [inline fe/src/components/Cities/CityCard.vue:5]: Carigara photo is not showing

**Review by LennardUy104 — CHANGES_REQUESTED** [review]
minor bug maybe choose different image for Carigara.

all good and working at my end

Squash commits

File: fe/src/components/Cities/CityCard.vue (hunk 1 above)
Comment (andreenriquez11) [reply fe/src/components/Cities/CityCard.vue:5]: Resolved

**Review by LennardUy104 — APPROVED** [review]
LGTM and working at my end nice!

File: fe/src/components/Navbar/NavigationBar.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <v-app-bar app>
-    <div class="title">SunBnB</div>
-    <input placeholder="Anywhere" class="search-bar" />
+    <div class="title"><a href="/">SunBnB</a></div>
+    <input placeholder="Anywhere" class="search-bar" v-model="searchInput" @keyup.enter="search" />
```
Comment (roseaugusto) [inline fe/src/components/Navbar/NavigationBar.vue]: you can enclose this properly in a form tag instead of checking the keyup

**Review by roseaugusto — CHANGES_REQUESTED** [review]

File: fe/src/components/Navbar/NavigationBar.vue (hunk 2 above)
Comment (andreenriquez11) [reply fe/src/components/Navbar/NavigationBar.vue]: changed it already maam

**Pull request merged**